INSTRUCTIONS.md

```diff
@@ -0,0 +1,129 @@
+# Introduction
+
+Hi there 👋,
+
+Thanks for taking the time to complete this brief take-home assignment. *Please
+timebox your effort to a maximum of 3 hours.*
+
+The goal of this exercise is to evaluate your frontend skills (React, styling,
+accessibility, responsive, 3rd-party integration, testing, and more). Feel free
+to make any assumptions, simplifications, or other changes to the problems -
+**though please state those in your write up (README.md) when you submit this
+assignment**. Please use as many libraries as is reasonable, however, please
+state your reasoning for such a library so we better understand your thought
+process.
+
+The objective of this assignment is to build a simple React application
+implementing a property list view using data returned from the SimplyRETS API
+and matching the following [Figma design
+mockup](https://www.figma.com/file/YZyIbis7fMsKnE2KaRlhYc/Sample-Project-Engineering?node-id=0%3A1).
+SimplyRETS is an API commonly used in real estate and you will need to use it to
+retrieve the properties data.
+
+Before getting started, please read this document carefully.
+
+**Good luck 🙃**
+
+# Acceptance criteria
+
+- Using the router *(included as a dependency)*, create the following page:
```

Comment:
Nit - to call out the actual tool used for routing in case they want quick access to docs

```suggestion
- Using [react-router](https://reactrouter.com/) *(included as a dependency)*, create the following page:
```